src/modules/drawer.js

```diff
@@ -387,6 +405,11 @@ Drawer.propTypes = {
     className: PropTypes.string,
     dimmer: PropTypes.bool,
     isOpen: PropTypes.bool.isRequired,
+    isSticky: PropTypes.bool,
```

Comment:
Can we change this `isSticky` prop to `positionY` and have it be an enum. `[ 'bottom', 'top']`. The default string for this would be `top`.
I suppose this messes with your top left corner radius logic too. Sorry about that. I would say if `maxHeight` isn't 100% and `positionX` is `top` the bottom left corner would be rounded and visa-versa for the top left corner.

Reply:
Fixed